index.html

```diff
@@ -1477,9 +1482,15 @@ <h2>
         </li>
 
         <li>
-Each service endpoint MUST include <code>id</code>,
-<code>type</code>, and <code>serviceEndpoint</code> properties, and
-MAY include additional properties.
+Each service endpoint SHOULD include an <code>id</code> property.
+The array of service endpoints MUST NOT contain duplicate entries
```

Comment:
```suggestion
The array of service endpoints MUST NOT contain multiple entries
```
... as above, because *duplicate* would be redundant, not conflicting.